constructors.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
```

Comment:
What is the purpose of this file vs. classtest.js? 

Since you aren't linking to this in your html, I'm going to hold off on leaving comments on this file. That said, if you don't need this file, you should get rid of it.